modules/library/jdbc/src/test/java/org/geotools/jdbc/SessionCommandListenerTest.java

```diff
@@ -78,7 +78,48 @@ public void testOnlyRelease() throws Exception {
     }
 
     @Test
-    public void testExpandVariables() throws Exception {
+    public void testTwoExpandVariablesAreReplaced() throws Exception {
+        SessionCommandsListener listener =
+                new SessionCommandsListener(
+                        "call startSession('select ${user}, ${version} from 1')", null);
+
+        // check borrow
+        EnvFunction.setLocalValue("user", "user1");
```

Comment:
While I realize the issue was already there, it's probably best to add a @After method that clears out the EnvFunction local variables, as the same JVM is used to run other tests (which may cause them to fail unexpectedly).